### PR TITLE
feat(description-list): Added bordered param to description-list - OEL-1777

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: src/compositions/bcl-description-list/description-list.html.twig [hunk 1]
```diff
@@ -31,8 +33,21 @@
   {% set attributes = create_attribute() %}
```
Comment (planctus) [inline src/compositions/bcl-description-list/description-list.html.twig:33]: Can we always apply a namespaced selector on the root element of every template..? bcl-description-list, in this case..?

File: src/compositions/bcl-description-list/description-list.html.twig [hunk 2]
```diff
@@ -44,21 +59,32 @@
...
+      >
+      {% if _item.term is iterable %}
+        <dt>
         {% for _term in _item.term %}
-          <dt
-          {% if _variant == 'horizontal' and _term.icon is not empty %}
-            class="d-flex"
-          {% endif %}
-          >
+            {% if _item.term.length > 1 %}
```
Comment (planctus) [inline src/compositions/bcl-description-list/description-list.html.twig:81]: is this working..? i know about a length filter in twig, not a "method"...

File: src/compositions/bcl-description-list/__snapshots__/description-list.test.js.snap [hunk 3]
```diff
@@ -41,114 +19,294 @@ exports[`OE - Description List horizontal renders correctly 1`] = `
...
+      <dd>
+        Description text goes here.
+      </dd>
+    </dl>
+    <dl
+      class="d-md-grid grid-3-9 mb-2-5"
+    >
+      <dt>
+        <div>
+
```
Comment (planctus) [inline src/compositions/bcl-description-list/__snapshots__/description-list.test.js.snap:34]: why this empty line..?

Comment (tibi2303) [reply src/compositions/bcl-description-list/__snapshots__/description-list.test.js.snap:34]: image

Cause of the bordered style, i needed to refactor the containers, but still every list has it's own dl tag, and all of them are wrapped inside this div container.

File: src/compositions/bcl-description-list/description-list.html.twig (hunk 1 above)
Comment (tibi2303) [reply src/compositions/bcl-description-list/description-list.html.twig:33]: done, should a ticket be created for all of these cases?

Comment (planctus) [reply src/compositions/bcl-description-list/description-list.html.twig:33]: i think so.. :)

**Comment by planctus** [comment]
can you just check the indentation in the template..? it's a bit off, as usual try to get html tags correctly indented, twig code is less important..

**Comment by tibi2303** [comment]
@planctus done.

**Review by planctus — APPROVED** [review]

**Pull request merged**